wagl/brdf.py

```diff
@@ -453,12 +501,53 @@ def only_ocean_pixels(src_poly, src_crs, fid_mask):
     return np.sum(ocean_mask & roi_mask) == 0
 
 
+def wbm_to_ocean_mask(result):
+    """
+    Converts an image from water-body mask (enum of 0-3) to land-ocean mask (bool).
+    """
+    OCEAN = 1
+    LAKE = 2
+    RIVER = 3
+    result = np.where(result == LAKE, OCEAN, result)
+    result = np.where(result == RIVER, OCEAN, result)
```

Comment:
If just creating a mask of "water = False, not water = True", why not simply do?:

```python
result = wbm == 0
```

A single call, that returns a bool.

Reply:
Haha good catch.